PrebidMobile/PrebidMobileRendering/Prebid/PBMCore/PBMPrebidParameterBuilder.m

```diff
@@ -81,6 +81,10 @@ - (void)buildBidRequest:(nonnull PBMORTBBidRequest *)bidRequest {
         bidRequest.extPrebid.cache = cache;
     }
     
+    if (adFormats.count >= 2) {
```

Comment:
Please comment on why we have this condition here and not just send the `includeformat` for all cases.